Simplenote/MetricsViewController.swift

```diff
@@ -82,16 +89,22 @@ class MetricsViewController: NSViewController {
 //
 private extension MetricsViewController {
 
-    func setupTextLabels() {
-        modifiedTextLabel.stringValue = NSLocalizedString("Modified", comment: "Note Modification Date")
-        createdTextLabel.stringValue = NSLocalizedString("Created", comment: "Note Creation Date")
-        wordsTextLabel.stringValue = NSLocalizedString("Words", comment: "Number of words in the note")
-        charsTextLabel.stringValue = NSLocalizedString("Characters", comment: "Number of characters in the note")
-    }
-
     func setupEntityObserver() {
         observer.delegate = self
     }
+
+    func setupResultsControllerIfNeeded() {
+        guard mustSetupResultsController, let plainInterlink = notes.first?.plainInterlink else {
+            return
+        }
+
+        resultsController.predicate = NSPredicate.predicateForNotes(exactMatch: plainInterlink)
+        try? resultsController.performFetch()
```

Comment:
We probably need to setup a listener here and update the table? 🤔

Reply:
🤦  I totally forgot about that. Done, thank you!!